unit_testing/cli/test_cli_sct_resample.py

```diff
@@ -1,14 +1,25 @@
-from pytest_console_scripts import script_runner
+
 import pytest
 import logging
 
+from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.scripts import sct_resample
+
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_resample_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_resample')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+@pytest.mark.parametrize("path_in,type_arg,dim,expected_dim", [
+    ('dmri/dmri.nii.gz', '-f', '0.5x0.5x1', (20, 21, 5, 7, 1.6826923, 1.6826923, 17.5, 2.2)),
+    ('t2/t2.nii.gz', '-mm', '0.97x1.14x1.2', (62, 48, 43, 1, 0.96774191, 1.1458334, 1.2093023, 1)),
+    ('t2/t2.nii.gz', '-vox', '120x110x26', (120, 110, 26, 1, 0.5, 0.5, 2.0, 1)),
+])
```

Comment:
Copy the "4d factor" etc comments down here

Reply:
00a61ca